crates/daemon/Cargo.toml

```diff
@@ -25,6 +25,7 @@ moor-db = { path = "../db" }
 moor-kernel = { path = "../kernel" }
 moor-var = { path = "../var" }
 rpc-common = { path = "../rpc/rpc-common" }
+tikv-jemallocator = "0.6.0"
```

Comment:
Version numbers go up in the root of the workspace, and then get used in the sub-crates with `.workspace = true`, take a look at the other dependencies for an example. I also try to keep the dependencies categorized.